Releasing.md

```diff
@@ -24,7 +24,7 @@ For the body of the post, just copy this checklist and again replace all occurre
 
 <details><summary>Click to expand</summary>
 <p>
-  
```

Comment:
Looks like VSCode removed some tailing whitespace. 
This wasn't intentional but seems fine to keep these changes in this PR